core/templates/dev/head/player/exploration_player.html

```diff
@@ -95,7 +96,9 @@ <h2 class="oppia-info-card-title"><[explorationCardTitle]></h2>
               src="/third_party/static/material-design-icons-1.0.1/ic_group_black_48dp.png">
             <li ng-repeat="ContributorUsername in explorationContributorUsernames| limitTo: 2"
               tooltip="<[ContributorUsername]>" tooltip-placement="top">
-              <img class="oppia-contributors-circle" ng-src="/images/avatar/user_mint_48px.png" alt="">
+              <profile-link-image linked-name="<[ContributorUsername]>">
+                <img class="oppia-contributors-circle" ng-src="/images/avatar/user_mint_48px.png" alt="">
```

Comment:
The img tag should probably go in the directive template; the directive should be able to get the image from the backend based on the username.